frontend/js/src/explore/layout.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import { Link, NavLink, Outlet, useLocation } from "react-router-dom";
+
+type NavbarLink = { to: string; label: string };
+
+const links: NavbarLink[] = [
+  { to: "fresh-releases/", label: "Fresh Releases" },
+  { to: "huesound/", label: "Huesound" },
+  { to: "cover-art-collage/", label: "Cover Art Collage 2023" },
+  { to: "cover-art-collage/", label: "Cover Art Collage 2023" },
+  { to: "cover-art-collage/2023/", label: "Cover Art Collage 2023" },
+  { to: "cover-art-collage/2022/", label: "Cover Art Collage 2022" },
+  { to: "music-neighborhood/", label: "Music Neighborhood" },
+  { to: "similar-users/", label: "Similar Users" },
+  { to: "lb-radio/", label: "LB Radio" },
+  { to: "similar-users/", label: "Similar Users" },
+  { to: "art-creator/", label: "Art Creator" },
+];
+
+function ExploreLayout() {
+  const location = useLocation();
+  const [activeLabel, setActiveLabel] = React.useState<string>("");
+
+  const getActiveLabel = React.useCallback((path: string) => {
+    const newActiveLabel = links.find((link) => path.includes(link.to))?.label;
```

Comment:
I'm new to this, so this is more a question than a comment:

Is `includes`  the right method to use here?
I'm thinking of `cover-art-collage/2023/` and `cover-art-collage/` above, which could lead to selecting the wrong link maybe?
If the path is `cover-art-collage/` it could result in selecting `cover-art-collage/2023/` or `cover-art-collage/2022/` if the order was different, which is pretty brittle.